tests/test__minhash.py

```diff
@@ -230,39 +234,26 @@ def test_size_limit(track_abundance):
     mh.add_hash(10)
     mh.add_hash(20)
     mh.add_hash(30)
-    assert mh.get_mins() == [10, 20, 30]
+    assert list(mh.hashes) == [10, 20, 30]
     mh.add_hash(5) # -> should push 30 off end
-    assert mh.get_mins() == [5, 10, 20]
-
-
-def test_max_hash(track_abundance):
-    # test behavior with max_hash
-    mh = MinHash(0, 4, track_abundance=track_abundance, max_hash=35)
-    mh.add_hash(10)
-    mh.add_hash(20)
-    mh.add_hash(30)
-    assert mh.get_mins() == [10, 20, 30]
-    mh.add_hash(40)
-    assert mh.get_mins() == [10, 20, 30]
-    mh.add_hash(36)
-    assert mh.get_mins() == [10, 20, 30]
+    assert list(mh.hashes) == [5, 10, 20]
 
 
 def test_scaled(track_abundance):
-    # test behavior with scaled (alt to max_hash)
-    scaled = get_scaled_for_max_hash(35)
-    print('XX', scaled, get_max_hash_for_scaled(scaled))
+    # test behavior with scaled
+    scaled = _get_scaled_for_max_hash(35)
+    print('XX', scaled, _get_max_hash_for_scaled(scaled))
     mh = MinHash(0, 4, track_abundance=track_abundance, scaled=scaled)
     assert mh.max_hash == 35
 
     mh.add_hash(10)
     mh.add_hash(20)
     mh.add_hash(30)
-    assert mh.get_mins() == [10, 20, 30]
+    assert mh.hashes == [10, 20, 30]
```

Comment:
hmm a bit surprised this still works, shouldn't it be list(mh.hashes)?

Reply:
indeed, it should be. this test was masked by another one below... and didn't work :)

fixed in 9d178c950fb45